dns_discover.go

```diff
@@ -14,7 +14,7 @@ const azURL = "http://169.254.169.254/latest/meta-data/placement/availability-zo
 
 var ErrNotInAWS = fmt.Errorf("Not in AWS")
 
-func discoverDNS(domain string, port int) (servers []string, ttl time.Duration, err error) {
+func discoverDNS(domain string, port int, context string) (servers []string, ttl time.Duration, err error) {
```

Comment:
- [ ] Again, "context" is an odd word here.  
  Instead, how about "urlBase?"

Reply:
Fixed, changed to 'urlBase'